### PR TITLE
Refer to version 3.x on download page

### PR DESCRIPTION
**Pull request by marcjansen**
Title says it all.

### CODE REVIEW

**Review by hblitza — APPROVED** [review]

**Comment by hblitza** [comment]
Thanks

**Comment by weskamm** [comment]
In my opinion we can remove the relation to a specific version. Maybe we need to release version 4.0.0 tomorrow? I think there is no need to have that information (current major) in the static docs

**Comment by marcjansen** [comment]
> In my opinion we can remove the relation to a specific version. Maybe we need to release version 4.0.0 tomorrow? I think there is no need to have that information (current major) in the static docs

What is your suggestion? Should I close this and you'll make another PR with what you think is more longliving?

File: src/pages/download.md [hunk 1]
```diff
@@ -10,11 +10,11 @@ There are several ready-to-use docker images for development/testing and product
...
 
-2.x is the current mainline of the development. All new features will be included in the next 2.x release (2.1.x, 2.2.x, etc.). We will make minor releases (e.g. 2.2.1) with small improvements and bugfixes as necessary.
+3.x is the current mainline of the development. All new features will be included in the next 3.x release (3.1.x, 3.2.x, etc.). We will make minor releases (e.g. 3.2.1) with small improvements and bugfixes as necessary.
 
 For changes between each release see the [CHANGES.txt of MapProxy](https://github.com/mapproxy/mapproxy/blob/master/CHANGES.txt).
 
-### 2.0.x
+### 3.0.x
 
 This is the current release. It is released under the [Apache Software License 2.0](http://www.apache.org/licenses/LICENSE-2.0.html).
```
Comment (weskamm) [inline src/pages/download.md:19]:
```suggestion
MapProxy is released under the [Apache Software License 2.0](http://www.apache.org/licenses/LICENSE-2.0.html).
```

**Review by weskamm — CHANGES_REQUESTED** [review]

**Comment by marcjansen** [comment]
Thanks for the suggestions, I accepted all of them.

Would you mind having another look?

**Comment by weskamm** [comment]
Nice, thanks!

**Pull request merged**